cognite/client/_api/ai/tools/documents.py

```diff
@@ -0,0 +1,114 @@
+from __future__ import annotations
+
+from collections.abc import Sequence
+
+from cognite.client._api_client import APIClient
+from cognite.client.data_classes.ai import Answer, AnswerLanguage, Summary
+from cognite.client.data_classes.data_modeling import NodeId
+from cognite.client.utils.useful_types import SequenceNotStr
+
+
+class AIDocumentsAPI(APIClient):
+    _RESOURCE_PATH = "/ai/tools/documents"
+
+    def summarize(
+        self,
+        ids: Sequence[int] | None = None,
+        external_ids: SequenceNotStr[str] | None = None,
+        instance_ids: Sequence[NodeId] | None = None,
+        ignore_unknown_ids: bool = False,
+    ) -> list[Summary]:
+        """Summarize a document using a Large Language Model.
+
+        Currently only supports summarizing a single document at a time, but
+        this may be extended in the future.
+
+        Args:
+            ids (Sequence[int] | None): Internal ids of documents to summarize.
+            external_ids (SequenceNotStr[str] | None): External ids of documents to summarize.
+            instance_ids (Sequence[NodeId] | None): Instance ids of documents to summarize.
+            ignore_unknown_ids (bool): Whether to skip documents that can't be summarized, without throwing an error
+
+        Returns:
+            list[Summary]: No description.
+        Examples:
+
+            Summarize a single document with internal id 123
+
+                >>> from cognite.client import CogniteClient
+                >>> client = CogniteClient()
+                >>> client.ai.tools.documents.summarize(ids=[123])
```

Comment:
May suggest we add an example using multiple as well?
```suggestion
                >>> client.ai.tools.documents.summarize(id=123)
                
            Summarize several documents:
            
                >>> from cognite.client.data_classes.data_modeling import NodeId
                >>> client.ai.tools.documents.summarize(
                ...     external_id=["foo", "bar"],
                ...     instance_id=NodeId("my-space", "file-xid"))
```